src/components/operations/operation-details/ko/runtime/type-definition.ts

```diff
@@ -62,11 +62,6 @@ export class TypeDefinitionViewModel {
         this.name(this.definition.name);
         this.description(this.definition.description);
         this.kind(this.definition.kind);
-
-        if (this.definition.example) {
-            this.exampleFormat(this.definition.exampleFormat);
-            this.example(this.definition.example);
-        }
```

Comment:
I don't think `exampleFormat` and `example` are used anywhere else. Maybe we could also remove their declarations?

Reply:
Indeed. Cleaned up.